HpToolsLauncher/Launcher.cs

```diff
@@ -879,30 +879,37 @@ private void RunTests(IAssetRunner runner, string resultsFile, TestSuiteRunResul
 
                 if (results.TestRuns.Count == 0)
                 {
-                    Console.WriteLine("No tests were run");
+                    ConsoleWriter.WriteLine(Resources.GeneralDoubleSeperator);
+                    ConsoleWriter.WriteLine("No tests were run");
                     _exitCode = ExitCodeEnum.Failed;
                     Environment.Exit((int)_exitCode);
                 }
 
                 //if there is an error
-                if (results.TestRuns.Any(tr => tr.TestState == TestState.Failed || tr.TestState == TestState.Error))
+                if (results.TestRuns.Any(tr => tr.TestState == TestState.Error))
                 {
```

Comment:
Why removing the `TestState.Failed` in the condition? The `Failed` state shall also return Failed exit code, no?